File: Python/String-2/double_char.py
"""
Given a string, return a string where for every char in the original, there are two chars.

double_char('The') → 'TThhee'
double_char('AAbb') → 'AAAAbbbb'
double_char('Hi-There') → 'HHii--TThheerree'

"""

def double_char(str):
  str1=""
  for i in range(len(str)):
    str1=str1+str[i]*2
  return str1

File: Java/String-2/doubleChar.java
/*
Given a string, return a string where for every char in the original, there are two chars.

doubleChar("The") ? "TThhee"
doubleChar("AAbb") ? "AAAAbbbb"
doubleChar("Hi-There") ? "HHii--TThheerree"
*/

public String doubleChar(String str) {
  String str1="";
  for(int i=0;i<str.length();i++){
    str1=str1+str.charAt(i)+str.charAt(i);
  }
  return str1;
}
